template/storybook/stories/Text.stories.tsx

```diff
@@ -1,7 +1,7 @@
 import { storiesOf } from '@storybook/react-native';
 import React from 'react';
 
-import { Text } from './Text';
+import { Text } from '../../src/components/Text';
```

Comment:
this feels a bit odd including `src` in the import, but not a big deal

Reply:
Yeah, I didn't want to move the storybook folder from the root level, do you think we should add it to src?

The reason why I did this is because I would do this with tests as well, tests would replicate the src directory in structure but have nothing but test files.